app/routes/crate.js

```diff
@@ -12,7 +12,7 @@ export default class CrateRoute extends Route {
     let crateName = params.crate_id;
 
     try {
-      return await this.store.findRecord('crate', crateName);
+      return await this.store.queryRecord('crate', { name: crateName });
```

Comment:
as mentioned in the issue, ideally we would redirect the user to the same route, but with the crate name replaced. this appears to be non-trivial though, since the user could also have requested a child route and we would potentially redirect them to the wrong route. I think this fix matches the previous behavior though, so it's probably good enough for now :)

Reply:
Fair enough! I have no idea why this would be somewhat non-trivial in Ember, especially for nested routes 😅 
Since `queryRecord` would make a request for every call, should we optimize this to something like `peekRecord` + `queryRecord`?

Reply:
> should we optimize this to something like `peekRecord` + `queryRecord`?

hmm, I guess that makes sense, yeah